src/device-registry/package.json

```diff
@@ -12,7 +12,10 @@
         "prod-pc": "SET NODE_ENV=production&&nodemon ./bin/www",
         "stage-pc": "SET NODE_ENV=staging&&nodemon ./bin/www",
         "prettier": "prettier --single-quote --print-width 80 --trailing-comma all --write 'src/**/*.js'",
-        "test": "mocha ./**/test/ut_*.js --exit"
+        "test": "mocha ./**/test/ut_*.js --exit",
+        "kafka-dev-mac": "NODE_ENV=development nodemon ./bin/kafka-connect.js",
```

Comment:
Yeah, this looks great. I gather that in the near future, we shall migrate this to a new microservice or just embed this inside the data-mgt/node microservice.

Reply:
Yeah, but it might cause code duplication. Here kafka is acting as an alternative to the insert measurements endpoint on device registry. The only difference is that instead of exposing an endpoint to accept measurements, it gets the measurements from a kafka topic, all the underlying validation and logic is the same. This means device registry has two entry points, kafka and the insert measurements endpoint. Attending to migrate kafka to the data-mgt folder means you have to copy the corresponding logic for inserting measurements and update it whenever you update device registry

Reply:
Cool, sounds reasonable...thanks. @NoahNsimbe